server/db/context/oauth_context.ts

```diff
@@ -6,8 +6,8 @@ import {
 } from "mysql2/promise";
 import { ServerError } from "../../middleware/errors";
 import pool from "../connect";
-import { mapDBToOAuthConnections } from "../mapper/oauth_mapper";
-import { TOAuthProvider } from "../model/oauth";
+import { mapDBToOAuthConnections } from "shared/community/oauth/oauth_mapper";
+import { TOAuthProvider } from "shared";
```

Comment:
nit: 두 import를 하나로 합칠 수 있습니다.

```suggestion
import { mapDBToOAuthConnections, TOAuthProvider } from "shared";
```